example-nextjs/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "yarn run link-react && next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "link-react": "cd ./node_modules/react && yarn link && cd ../../../packages/run-wasm && yarn link \"react\""
```

Comment:
Since we're using a local linked module for `run-wasm`, we have to run this script to tell the `run-wasm` module to use the `example-nextjs` project's copy of react. Users using the `run-wasm` npm package won't have to do this linking because the peer dependency should Just Work.